lessons/12_scope/index.md

```diff
@@ -0,0 +1,148 @@
+# Lecture 12 Variable Scope
```

Comment:
are you planning to add regular slides?

Reply:
Plan is to present the html page and scroll through it explaining stuff. Not feeling benefit of the slides for this lecture. We post a link so they can read ahead on slide if they want anyway.

Reply:
Yes, plan is clear but since we're using slides => it should be slides at the end. Content should be consistent.

Reply:
I was unaware where it is written we have to create slides using the existing html template. It is much faster for me to create markdown files.